Makefile

```diff
@@ -5,7 +5,7 @@ DESTINATION = /usr/local/lib/pam
 TARGET = x86_64-apple-macosx10.12
 
 all:
-	$(XCODE)/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc touchid-pam-extension.swift -o $(LIBRARY_NAME) -target $(TARGET) -emit-library -sdk $(XCODE)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk
+	swiftc touchid-pam-extension.swift -o $(LIBRARY_NAME) -target $(TARGET) -emit-library -sdk $(XCODE)/SDKs/MacOSX10.12.sdk
```

Comment:
The `-sdk` path here is incorrect. `SDKs` are deep into

`Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer`

and `xcode-select -p` gives you `Xcode.app/Contents/Developer`

Reply:
We can potentially use ` $(XCODE)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk` instead but I didn't intentionally just in case future SDKs are unsupported